external/storm-kafka-monitor/src/main/java/org/apache/storm/kafka/monitor/KafkaOffsetLagUtil.java

```diff
@@ -89,6 +89,10 @@ public static void main (String args[]) {
                     printUsageAndExit(options, OPTION_ZK_SERVERS_LONG + " and " + OPTION_ZK_COMMITTED_NODE_LONG + " are required  with " +
                             OPTION_OLD_CONSUMER_LONG);
                 }
+                String zkNode = commandLine.getOptionValue(OPTION_ZK_COMMITTED_NODE_LONG);
```

Comment:
`zkNode == null` is not unnecessary since `hasArg` is set to true. and I will remove.
but `zkNode.length() <= 1` is necessary because zkPath should start with `/`, so the valid path length should `> 1`.